lib/gs-app-list-private.h

```diff
@@ -45,16 +45,13 @@ typedef enum {
  * Flags to use when filtering. The priority of eash #GsApp is used to choose
```

Comment:
If submitting this change upstream, `s/eash/each/` here (I know it's existing code)

Reply:
Done in this PR by Andre, and in the upstream patch (also by Andre) here: https://gitlab.gnome.org/GNOME/gnome-software/merge_requests/343